src/sas/sascalc/calculator/slit_length_calculator.py

```diff
@@ -80,19 +80,19 @@ def calculate_slit_length(self):
                 break
 
         # y value and ind just before passed the spot of the half height
-        y_half_u = y[ind-1]
+        y_half_u = y[ind - 1]
 
         # get corresponding x values
         x_half_d = x[ind]
-        x_half_u = x[ind-1]
+        x_half_u = x[ind - 1]
 
         # calculate x at y = y_half using linear interpolation
         if y_half_u == y_half_d:
             x_half = (x_half_d + x_half_u)/2.0
```

Comment:
The only possible improvement I see is to add whitespace around the division sign here. Otherwise, everything else looks okay to me.